SYCL/ESIMD/api/functional/ctors/esimd_ctors_common.hpp

```diff
@@ -0,0 +1,55 @@
+//===-- esimd_ctors_common.hpp - Define common code for simd ctors tests --===//
```

Comment:
```suggestion
//===-- common.hpp - Define common code for simd ctors tests --===//
```
See, you already have "esimd" and "ctors" in the path: `ESIMD/.../ctors/common.hpp`, so why duplicate?

Reply:
this comment was applied in this commit: 3e7019d1753cffdbe4a35aafaa674d765639be0e